test/pytest/test.py

```diff
@@ -719,6 +719,10 @@ def testDoubleParse(self):
         self.assertEqual(1512060373.222988, float(res))
         # self.assertEqual('1512060373.222988', res)
 
+    def testDebugMemory(self):
+        self.cmd('JSON.SET', 'memoryTest', '.', '{"key1":"val", "key2":[1,2,3], "key3":{"key4":1,"key5":false}}')
+        res = self.cmd('JSON.DEBUG', 'MEMORY', 'memoryTest')
+        self.assertEqual(439.0, float(res))
```

Comment:
Tests are always good, but tying this to the implementation is bad. I would just check > 0 (and maybe < 1000).